### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by yorekKap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -425,6 +425,82 @@ compile group: 'com.github.corese4rch', name: 'cvurl-io', version: '1.1'
...
+ 2) **More convenient and stylish configuration management**   
+     First I would like to give a little bit of introduction to what can look like not a 
+     large switch but in reality will be a great asset to the ease of using and developing CVurl.
+     
+     So what we tried to achieve introducing those changes:
+      
+    * **Better scoping**    
+    We provided clean separation of configuration scopes providing general per CVurl instance CVUrlConfiguration and more specific per request RequestConfiguration. CVurlConfiguration contains central gears to maintain our wrapper such as 
+    HttpClient, GenericMapper and so on. RequestConfiguration on the other hand is newly created on each request and serves as context for different properties of requests such as requestTimeout, acceptCompressed and so on.I like to think about this way:
+        * Single CVurlConfiguration is per single instance of CVurl.
```
Comment (andyislegend) [inline README.md]: One CVurlConfiguration per single instance of CVurl

File: README.md [hunk 2]
```diff
@@ -425,6 +425,82 @@ compile group: 'com.github.corese4rch', name: 'cvurl-io', version: '1.1'
...
+     
+     So what we tried to achieve introducing those changes:
+      
+    * **Better scoping**    
+    We provided clean separation of configuration scopes providing general per CVurl instance CVUrlConfiguration and more specific per request RequestConfiguration. CVurlConfiguration contains central gears to maintain our wrapper such as 
+    HttpClient, GenericMapper and so on. RequestConfiguration on the other hand is newly created on each request and serves as context for different properties of requests such as requestTimeout, acceptCompressed and so on.I like to think about this way:
+        * Single CVurlConfiguration is per single instance of CVurl.  
+        * CVurlConfiguration as well as central components of the system contains instance of RequestConfiguration which values will be used as default properties of RequestConfiguration created from this CVurl. On each sending of actual request (asObject(), asString(), ...) new instance of RequestConfiguration will be created with default properties provided by central global RequestConfiguration from CVurlConfiguration.So...
```
Comment (andyislegend) [inline README.md]: Configuration, as well as central components of the system,

File: README.md [hunk 3]
```diff
@@ -425,6 +425,82 @@ compile group: 'com.github.corese4rch', name: 'cvurl-io', version: '1.1'
...
+      //now you can do it with no fuss
+      List<User> resultUsers = cvurl.get(url)
+              .asObject(new BodyType<List<User>>() {}, HttpStatus.OK)
+              .orElseThrow(RuntimeException::new);
+    } 
+    ```      
+
+ 2) **More convenient and stylish configuration management**   
+     First I would like to give a little bit of introduction to what can look like not a 
+     large switch but in reality will be a great asset to the ease of using and developing CVurl.
```
[the remark this answers is not in the document]
Comment (yorekKap) [reply README.md]: using with cvurl ? using and developing cvurl - meaning users using it and we developers developing it.

[the remark this answers is not in the document]
Comment (andyislegend) [reply README.md]: aa, slightly diff meaning I get from this sentence. Then, we need to rewrite it. 
- 'to the ease of use for developers and ease for further development for us'

File: README.md [hunk 4]
```diff
@@ -450,8 +450,8 @@ compile group: 'com.github.corese4rch', name: 'cvurl-io', version: '1.1'
     * **Better scoping**    
     We provided clean separation of configuration scopes providing general per CVurl instance CVUrlConfiguration and more specific per request RequestConfiguration. CVurlConfiguration contains central gears to maintain our wrapper such as 
     HttpClient, GenericMapper and so on. RequestConfiguration on the other hand is newly created on each request and serves as context for different properties of requests such as requestTimeout, acceptCompressed and so on.I like to think about this way:
-        * Single CVurlConfiguration is per single instance of CVurl.  
-        * CVurlConfiguration as well as central components of the system contains instance of RequestConfiguration which values will be used as default properties of RequestConfiguration created from this CVurl. On each sending of actual request (asObject(), asString(), ...) new instance of RequestConfiguration will be created with default properties provided by central global RequestConfiguration from CVurlConfiguration.So...
+        * One CVurlConfiguration is per single instance of CVurl.
```
Comment (andyislegend) [inline README.md]: 'is' is redundant

**Pull request merged**